pay.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+require('config.php');
+require('razorpay-php/Razorpay.php');
+session_start();
+
+// Create the Razorpay Order
+
+use Razorpay\Api\Api;
+
+$api = new Api($keyId, $keySecret);
+
+$orderData = [
+    'receipt'         => 3456,
+    'amount'          => 2000 * 100, // 2000 rupees in paise
```

Comment:
Have a variable for $amount instead, so it is easier to understand?
receipt -> explain what the variable does?

leave as many comments as you can...